pages/integrations/integrations.mdx

```diff
@@ -0,0 +1,10 @@
+---
+layout: 'guide'
+title: 'All integrations'
+---
+
+import Link from '../../components/Link';
+
+Here is a list of tools you can integrate Avo with
+
+<Link passHref href="/integrations/segment"><a>Segment</a></Link>
```

Comment:
Here will be a perfect opportunity to highlight and explain the different integration types we offer (productivity, observability, implementation etc) and list out the tools that support each